docker-compose.yml

```diff
@@ -0,0 +1,167 @@
+version: '3.8'
+services:
+  # Run webpack-dev-server and compile static assets.
+  #
+  # This is the primary server and it proxies the
+  # Django app so that Webpack can watch for file changes.
+  assets:
+    build:
+      context: .
+      target: assets
+    image: mozmeao/birdbox_assets:${GIT_COMMIT:-latest}
+    platform: linux/amd64
+    command: npm run watch
+    ports:
+      - "8000-8010:8000-8010"
+    volumes:
+      - ./src/:/app/src:delegated
+      - ./birdbox/:/app/birdbox:delegated
+      - ./assets/:/app/assets:delegated
+    environment:
+      WP_PROXY_URL: "http://app:8080"
+
+  # The Django app
+  app:
+    build:
+      context: .
+      target: devapp
+    command: python birdbox/manage.py runserver 0.0.0.0:8080
+    depends_on:
+      - db
+      - redis
+    env_file: ./docker/envfiles/local.env
+    environment:
+      DATABASE_URL: postgres://postgres:postgres@db/postgres
+      REDIS_URL: redis://redis:6379
+    image: mozmeao/birdbox_test:${GIT_COMMIT:-latest}
+    platform: linux/amd64
+    ports:
+      - "8080:8080"
+    volumes:
+      - ./birdbox/:/app/birdbox:delegated
+      - ./bin/:/app/bin:delegated
+      - ./local-credentials/:/app/local-credentials:delegated
+      - ./docker/:/app/docker:delegated
+      - ./etc/:/app/etc:delegated
+      - ./src/:/app/src:delegated
+      - ./root_files/:/app/root_files:delegated
+      - ./wsgi/:/app/wsgi:delegated
+      - ./assets/:/app/assets:delegated
```

Comment:
Yeah i think bedrock is explicit about what it mounts and does not. For instance we wouldn't want to mount birdbox/media from the host into the docker container, because then you'd have images stored on the host, but the data that knows about those images is in the DB that's in the Dockerverse. 

I agree, though, that what i've got here is pretty verbose